### ISSUE TITLE
Adjust WDL to accomodate SRA data, specifically bbtools flag `filterbytile`

### ISSUE THREAD

**Issue by samobermiller** [microbiomedata/ReadsQC#65]
Need to be able to turn `filterbytile` from bbtools off when processing SRA data. In bbtools 39.65 it is on by default and it causes our workflow to fail (example jaws ids 168329, 168330 and 168332 from PR #63) producing threading errors when headers are parsed. @aclum
